### ISSUE TITLE
Wildcards cannot be used as the first character

### ISSUE THREAD

**Issue by falk-22** [MicrosoftDocs/office-docs-powershell#8941]
One important information is missing on this page: Wildcards cannot be used as the first character.
This makes it impossible to filter for SMTP addresses or UPNs, etc.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3762ada1-99d6-fa6d-fbd7-524003f8c3b2
* Version Independent ID: 03e801b7-fa14-b5fb-8600-131c9e73292f
* Content: [Filterable properties for the RecipientFilter parameter](https://docs.microsoft.com/en-us/powershell/exchange/recipientfilter-properties?view=exchange-ps)
* Content Source: [exchange/docs-conceptual/recipientfilter-properties.md](https://github.com/MicrosoftDocs/office-docs-powershell/blob/master/exchange/docs-conceptual/recipientfilter-properties.md)
* Service: **exchange-powershell**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

**Comment by chrisda** [comment]
@falk-22, you are correct for filters in the cloud. The last time I checked (which, admittedly, has been a while), on-prem Exchange doesn't have this limitation. The topic was originally written for on-prem Exchange, and it was later (a couple of years ago) made single-source for both cloud and on-prem.

I'll make the update and close this issue when I'm done.

**Comment by chrisda** [comment]
@falk-22, fixed.

Thanks for the feedback.

Closing
